### ISSUE TITLE
Tooltip causes console errors when wrapped around CheckboxButton

### ISSUE THREAD

**Issue by StephenHaney** [modulz/radix#166]
![image](https://user-images.githubusercontent.com/4045376/64728567-1db08e80-d490-11e9-8069-1ce5c9e99cde.png)

Here's the code that's throwing the error:
```
    <Tooltip label={tooltipLabel} align={tooltipAlignment}>
      <Checkbox checked={Boolean(checked) || false} onChange={handleChange} />
    </Tooltip>
```

**Comment by peduarte** [comment]
Oh, this one again! I thought I added `forwardRef` around all the components. Is this with the latest version of Radix (`@modulz/radix@0.0.1-alpha.50
`)?

**Comment by StephenHaney** [comment]
No, this is on the `develop` branch of `modulz/modulz`... I tried to update but it looks like some other things have changed (no more Pill?).
